NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Ticket 39- Unapproved Posts

Description
Added list of unapproved posts. When author's create posts they require approval before being displayed. Admins do not need approval. [Trello Ticket #39](https://trello.com/c/97SnkgSd)

Type of Change
-Added new, non-breaking feature.

Testing Instructions


1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout sf-ticket-39
5. If you don't see the most recent version of my files, git pull origin sf-ticket-39
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in you client folder. Run "npm start" in your terminal.
11. Login as an author. rdo1@timesonline.co.ukx for the email parameter. Leave the password blank. Click "Login."
12. 
Click "Create New Post. " Create a new post titled "Waiting for Approval" with a publish date of yesterday. Click Create.
13. You should be redirected to the "All Posts" page and you should NOT see your post.
14. Click on "My Posts" and you should see your newly created post there.
15. Login as an admin. [foo@bar.com](mailto:foo@bar.com) for the email parameter. Leave the password text box blank. Click "Login".
16. Repeat steps 12 and 13, but this time you SHOULD see your post. 
17. Click on "Unapproved Posts" in the navbar and you should see a list of all unapproved posts.

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
